clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ConsumerActionType.java

```diff
@@ -9,7 +9,7 @@ public enum ConsumerActionType {
    * KILL action has higher priority than others, so that once KILL action is added to the action queue,
    * we will process it immediately to avoid doing throw-away works.
    */
-  KILL(2), STANDBY_TO_LEADER(1), LEADER_TO_STANDBY(1);
+  KILL(2), STANDBY_TO_LEADER(1), LEADER_TO_STANDBY(1), VERSION_ROLE_CHANGE_TO_TRIGGER_RESUBSCRIBE(1);
```

Comment:
I feel `VERSION_ROLE_CHANGE_TO_TRIGGER_RESUBSCRIBE` is too specific as it includes concrete action into the `type`, and the concrete actions can change in the future, so I feel `VERSION_ROLE_CHANG` is good enough.